src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -47,13 +48,14 @@ class IOUAmountPage extends React.Component {
 
         this.updateAmountIfValidInput = this.updateAmountIfValidInput.bind(this);
         this.state = {
-            amount: '',
+            amount: props.selectedAmount || '',
         };
     }
 
     componentDidMount() {
         if (this.textInput) {
-            this.textInput.focus();
+            // Input won't focus unless a setTimeout is set with atleast 300
+            setTimeout(() => this.textInput.focus(), 300);
```

Comment:
We were already trying to focus the input here but it wasn't working - the only workaround that worked was adding a setTimeout for anything > 300.  I also tried the autoFocus prop on the textInput but that didn't work either